.buildscript/e2e.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+if [ "$RUN_E2E_TESTS" != "true" ]; then
+  echo "Skipping end to end tests."
+else
+  echo "Running end to end tests..."
+  wget https://github.com/segmentio/library-e2e-tester/releases/download/0.2.1/tester_linux_amd64
+  chmod +x tester_linux_amd64
+  ./tester_linux_amd64 -segment-write-key="$SEGMENT_WRITE_KEY" -webhook-auth-username="$WEBHOOK_AUTH_USERNAME" -webhook-bucket="$WEBHOOK_BUCKET" -path='./cli.js'
```

Comment:
It's an edge case but please note that if `tester_linux_amd64` gets killed by a signal (SIGKILL, SEGSEV, etc) this will leak the value of the arguments. Example with a segfault : 
```
take /tmp/test
echo "void main() { *((int*)(void*)0xdeadbeef) = 1; }" > test.c
gcc test.c -o test -w
./test --write-key=secretstuff
```

You should see `[1]    <PID> segmentation fault  ./test --write-key=secretstuff`.


It'd be safer if the tester could just read the environment variables.

Reply:
updated to use env vars!